README.md

```diff
@@ -176,4 +177,31 @@ Space: O(1)
 - Nic Paro
 
 #### Solution
-![Whiteboard](code401challenges/assets/linked-list-merge.jpg)
\ No newline at end of file
+![Whiteboard](code401challenges/assets/linked-list-merge.jpg)
+
+--------------------------------------------------------------
+
+### 9. Stacks and Queues
+
+#### Challenge
+Implement a Stack and a Queue Data Structure
+
+#### Approach & Efficiency
+Create a Node class that has properties for the value stored in the Node, and a pointer to the next node.  For the Stack class implement methods for push, pop, and peek.  For the Queue class implement mehtods for enqueue, dequeue, and peek.
+
+Big O Notation
```

Comment:
You should list the Big O of each method! If the Big O is the same for everything then you should make sure it says that in the README